homeassistant/components/minecraft_server/const.py

```diff
@@ -25,6 +26,7 @@
 NAME_PROTOCOL_VERSION = "Protocol Version"
 NAME_STATUS = "Status"
 NAME_VERSION = "Version"
+NAME_MOTD = "World Message"
```

Comment:
Do people call the MOTD a world message?

Reply:
I've seen it called a lot of things.

Reply:
Okay, I'm good with it then.